bundles/org.eclipse.ui.genericeditor/plugin.xml

```diff
@@ -260,4 +260,19 @@
             contentType="org.eclipse.core.runtime.text">
       </reconciler>
    </extension>
+   <extension
```

Comment:
Is there a way to hide this UI contribution for certain file types in a product?

Reply:
Oh, I misread. I was under the assumption that this is also shown in the `Open With` but that's not the case, is it?

Reply:
> Oh, I misread. I was under the assumption that this is also shown in the `Open With` but that's not the case, is it?

Nope.